jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsListViewImpl.java

```diff
@@ -38,6 +42,7 @@
 @Dependent
 @Templated
 public class CaseActionsListViewImpl extends AbstractView<CaseActionsPresenter> implements CaseActionsPresenter.CaseActionsListView {
+    private int PAGE_SIZE = 4;
```

Comment:
This is definitely a good place to add clarification for the case worker about where the Ad-hoc actions are located in the model.
The same three statuses come to my mind too.
My two pennyworth when it comes to status names:
- Dynamic (would also drop "Add new dynamic" from the user/process task names)
- Available in: Stage name
- Available in: Case